src/app/components/nav/header-static/header-static.component.html

```diff
@@ -9,6 +9,7 @@
         <!-- show after authenticate user -->
         <li *ngIf="authService.isAuth && !isDash"><a class="waves-effect waves-dark main-header-link" (click) ="isDash = true" routerLink="/dashboard" routerLinkActive="active">Dashboard</a></li>
         <li *ngIf="authService.isAuth && isDash"><a class="waves-effect waves-dark main-header-link" routerLink="/challenges" routerLinkActive="active">All Challenges</a></li>
+        <li *ngIf="authService.isAuth"><a class="waves-effect waves-dark main-header-link" href="https://evalai.readthedocs.io/en/latest/" target="_blank">Docs</a></li> 
```

Comment:
Why this change?

Reply:
I was comparing the original evalai with evalai-ngx and found that the there is a link to the documentation in the navbar of original evalai code. So, i added this line to for the same functionality in evalai-ngx too.

Reply:
Ok but this isn't related to this pr so undo it please

Reply:
Okay. I will create a different pr for this!